### DISCUSSION TITLE
Does the application have a little Turkish language support?

### DISCUSSION THREAD

**Discussion by AhmetPolat43** [OpenOrienteering/mapper#1805] [Questions & Answers]
Does the application have a little Turkish language support?

**Comment by dg0yt** [comment]
So far, the Turkish translation is at 15% for the Mapper program. That's why it is not yet included in the packages.

Contribution is possible online:
https://hosted.weblate.org/projects/openorienteering/

**Reply by AhmetPolat43** [reply]
I will look at the translation part as soon as possible. I've asked before, but Turkey also use the openorienteering mapper, I am preparing a series of training to improve the recognition and share it'll be no problem at diym of Udemy?

**Reply by dg0yt** [reply]
> I am preparing a series of training to improve the recognition and share it'll be no problem at diym of Udemy?

OpenOrienteering Mapper can be shared without asking us for permission. This is part of the GPL-3 license.

However, I suggest that you just share our download links if possible. In this case, you don't have to worry about some obligations which this license does create if you distribute our binary packages.
